### PR TITLE
Akka projection mongo

### PR DESCRIPTION
**Pull request by jtjeferreira**
Integration with mongoDB:
* MongoDB changestream as source of events
* MongoDB as offset storage

### CODE REVIEW

File: examples/src/test/scala/docs/kafka/KafkaDocExample.scala [hunk 1]
```diff
@@ -18,7 +18,7 @@ import akka.kafka.scaladsl.SendProducer
 import akka.projection.Projection
 import akka.projection.ProjectionBehavior
 import akka.projection.ProjectionId
-import akka.projection.kafka.GroupOffsets
+import akka.projection.mongo.GroupOffsets
```
Comment (ignasi35) [inline examples/src/test/scala/docs/kafka/KafkaDocExample.scala:21]: These changes seem like an unwanted side-effect of a mass-search and replace.

**Comment by patriknw** [comment]
Thanks for contributing @jtjeferreira 
This project is still in early development with big refactorings going on all the time. We will not be able to bring in more source code until that has stabilized. Probably within 1-2 months.
We will let you know when we are prepared to accept bigger contributions like this. I hope you can understand.

**Comment by jtjeferreira** [comment]
@patriknw no worries, I will try to rebase it from time to time

**Comment by octonato** [comment]
Hey @jtjeferreira, 

Now that we have released version 1.0 and that the APIs are more stable, it's a good moment to resume work on this PR. If you are still interested and have the time of course.

We will probably want to make a distinction between officially supported modules and community maintained ones. Maybe a banner on the MongoDB Projection documentation.

Does that work for you?

**Comment by patriknw** [comment]
closing this due to inactivity

**Pull request closed without merging**